docs/examples/schema_extra_callable.py

```diff
@@ -8,8 +8,10 @@ class Person(BaseModel):
 
     class Config:
         @staticmethod
-        def schema_extra(schema: Dict[str, Any]) -> None:
-            for prop in schema.get('properties', {}).values():
-                prop.pop('title', None)
+        def schema_extra(
+            schema: Dict[str, Any], model_class: Type["Person"]
+        ) -> None:
+            for prop in schema.get("properties", {}).values():
+                prop.pop("title", None)
```

Comment:
```suggestion
                prop.pop('title', None)
```
pydantic uses single quotes be default.